meta-oe/recipes-support/c-ares/c-ares_1.19.1.bb

```diff
@@ -5,12 +5,8 @@ SECTION = "libs"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://LICENSE.md;md5=fb997454c8d62aa6a47f07a8cd48b006"
 
-SRC_URI = "git://github.com/c-ares/c-ares.git;branch=main;protocol=https \
-           file://CVE-2022-4904.patch \
-           file://CVE-2023-31130.patch \
-           file://CVE-2023-32067.patch \
-          "
-SRCREV = "2aa086f822aad5017a6f2061ef656f237a62d0ed"
+SRC_URI = "git://github.com/c-ares/c-ares.git;branch=main;protocol=https"
+SRCREV = "6360e96b5cf8e5980c887ce58ef727e53d77243a"
```

Comment:
It looks like https://github.com/ni/meta-openembedded/commit/5adcef034848103a579bcac9c9ff379ac4ed6087 may also be redundant in 1.19.1.

Reply:
Pushed a revert.